rstest/tests/fixture/mod.rs

```diff
@@ -5,6 +5,14 @@ use super::resources;
 use mytest::*;
 use rstest_test::{assert_in, assert_not_in, Project, Stringable, TestResults};
 
+const SRC_LIB_RS: &str = {
+    const SEP: u8 = std::path::MAIN_SEPARATOR as u8;
+    const SRC_LIB_RS: [u8; 11] = [
+        SEP, b's', b'r', b'c', SEP, b'l', b'i', b'b', b'.', b'r', b's',
+    ];
+    unsafe { std::str::from_utf8_unchecked(&SRC_LIB_RS) }
+};
+
```

Comment:
Sure there isn't any simpler way to make a portable static path?  Maybe use `OnceLock` or  a function to get it is better, it's test code: performance is a detail against readability.